src/MicroflowTimer/widget/MicroflowTimer.js

```diff
@@ -161,6 +163,16 @@ define([
             }
         },
 
+        _executeEvent: function() {
+            if(this.callEvent === "callMicroflow" && this.microflow) {
+                this._execMf()
+            } else if (this.callEvent === "callNanoflow" && this.nanoflow){
+                this._executeNanoFlow()
+            } else {
+                mx.ui.error("No action specified for " + this.callEvent)
```

Comment:
To late, should be mentioned in postCreate

Reply:
done